tests/acceptance/common.py

```diff
@@ -268,51 +268,53 @@ def common_boot_from_internal(conn):
 
 def latest_build_artifact(builddir, extension, sdimg_location=None):
     global configuration
-    
+
     if configuration.get("conversion"):
-        output = subprocess.check_output(["sh", "-c", "ls -t %s/%s/*%s | grep -v data*%s| head -n 1" % (builddir, sdimg_location, extension, extension)])
+        output = subprocess.check_output(["sh", "-c", "ls -t %s/*%s | grep -v data*%s| head -n 1" % (builddir, extension, extension)])
     else:
         output = subprocess.check_output(["sh", "-c", "ls -t %s/tmp*/deploy/images/*/*%s | grep -v data*%s| head -n 1" % (builddir, extension, extension)])
     output = output.decode().rstrip('\r\n')
-    print("Found latest image of type '%s' to be: %s" % (extension, output))
-    return output
+    path_to_image = os.path.abspath(output)
+    print("Found latest image of type '%s' to be: %s" % (extension, path_to_image))
+    return path_to_image
```

Comment:
@pasinskim FYI this is the change that makes the tests fail. Probably the working dir is not what I assumed to be. I don't have more time today but I'll look into it tomorrow.

Reply:
Reworked